pkg/karmadactl/register/register.go

```diff
@@ -90,8 +90,14 @@ const (
 	// KarmadaAgentKubeConfigFileName defines the file name for the kubeconfig that the karmada-agent will use to do
 	// the TLS bootstrap to get itself an unique credential
 	KarmadaAgentKubeConfigFileName = "karmada-agent.conf"
-	// KarmadaKubeconfigName is the name of karmada kubeconfig
-	KarmadaKubeconfigName = "karmada-kubeconfig"
+	// KarmadaConfigSecretName is the secret name of karmada config for agent
+	KarmadaConfigSecretName = "karmada-agent-config" //nolint:gosec
```

Comment:
actually no any sensitive information

`gosec` nolint is required just because the variable name `KarmadaConfigSecretName` contains the word `Secret`.

However, I don't want to rename to other name, I think the current variable name is the most accurate.